packages/angular/cli/lib/examples/signal-forms/15-accessible-styled-forms.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Accessible and Styled Signal Forms
+summary: Demonstrates how to build a production-ready signal form that is accessible to screen reader users and provides clear visual feedback using CSS and class bindings.
+keywords:
+  - signal forms
+  - accessibility
+  - a11y
+  - styling
+  - css
+  - aria
+  - class binding
+required_packages:
+  - '@angular/forms'
+related_concepts:
+  - 'signals'
+  - 'accessibility'
+experimental: true
+---
+
+## Purpose
+
+The purpose of this pattern is to create forms that are usable by everyone, including people with disabilities, and provide a clear, intuitive user experience. It solves the problem of building forms that not only work correctly but are also compliant with accessibility standards (a11y) and provide visual cues for different states (e.g., invalid, touched).
+
+## When to Use
+
+Apply these patterns to all forms in a production application. Accessibility is a requirement for modern web development, and clear visual styling for form states is a fundamental aspect of good user experience design.
+
+## Key Concepts
+
+- **`[class.invalid]`:** A class binding that applies a CSS class to an element when a condition is true. Here, it's used to style an input based on its validity.
```

Comment:
a class is not really required here. The attribute selector [aria-invalid] can be used for styling it as an error.